listenbrainz/webserver/static/js/src/LastFMImporter.test.tsx

```diff
@@ -133,6 +133,7 @@ describe("getTotalNumberOfScrobbles", () => {
 });
 
 describe("getPage", () => {
+  const RETRIES = 3;
```

Comment:
I think we can export this variable from the Importer file and then use it here. In case we change the value in the future, this might prevent tests from breaking.